src/components/books/AddBook.js

```diff
@@ -1,12 +1,104 @@
+// import React, { useState } from 'react';
+// // eslint-disable-next-line import/no-extraneous-dependencies
+// import PropTypes from 'prop-types';
+// import classes from './AddBook.module.css';
+
+// const AddBook = ({ onNewBook }) => {
+//   const [title, setTitle] = useState('');
+//   const [author, setAuthor] = useState('');
+//   const [gunra, setGunra] = useState('');
+
+//   const titleChangeHandler = (event) => {
+//     setTitle(event.target.value);
+//   };
+
+//   const authorChangeHandler = (event) => {
+//     setAuthor(event.target.value);
+//   };
+
+//   const gunraChangeHandler = (event) => {
+//     setGunra(event.target.value);
+//   };
+
+//   const submitChangeHandler = (event) => {
+//     event.preventDefault();
+
+//     if (title.trim().length === 0 || author.trim().length === 0 || gunra.trim().length === 0) {
```

Comment:
- You have done great so far :+1: :+1: Your app meets all the requirements and works as expected. but there are a few things you need to correct before moving to the next milestone. :smile: :wave: 

- [ ] Following the best practices, you are not meant to leave unused code as a comment on your code base. it makes your codebase look unprofessional. I'll suggest you remove all these commented-out codes . :smile: :wave: